.github/workflows/monitoring.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - sm/gha
+  workflow_dispatch:
+  schedule:
+    - cron: 7 4 * * *
```

Comment:
```suggestion
    - cron: */50 * * * * # every 50 minutes
```

Reply:
that format is not supported.  gha uses only posix cron (very basic).

It's back to hourly